macros/edr/system/system_utils/timestamp_column.sql

```diff
@@ -4,4 +4,8 @@
 
 {% macro current_timestamp_column() %}
     cast ({{elementary.current_timestamp_in_utc()}} as {{ elementary.type_timestamp() }})
-{% endmacro %}
\ No newline at end of file
+{% endmacro %}
+
+{% macro formatted_current_timestamp_column() %}
```

Comment:
Why not renaming this as well and moving it to times.sql?

Reply:
Because this method is specific for creating columns and use inside of a query (just like run_start_column and current_timestamp_column).
It is not a generic time macro that is used to get the current time as datetime/str.

This is why the name fits.
It is current_timestamp_column with a format.